std/format.d

```diff
@@ -4300,9 +4300,9 @@ T unformatValue(T, Range, Char)(ref Range input, ref FormatSpec!Char spec)
 {
     import std.algorithm.searching : find;
     import std.conv : parse, text;
-    if (spec.spec == 's')
+    static if (is(typeof(parse!T(input)) == T))
```

Comment:
So here `T` is always a (possibly qualified) `bool`. Under what circumstances may `parse!T(input)` not compile?